src/PlayerStrategies/PlayerStrategies.cpp

```diff
@@ -289,34 +318,6 @@ void HumanPlayer::issueOrder(std::vector<Player*>* players) {
             std::string targetTerritory;
             int numOfUnits = 0;
 
-            std::cout << "From which territory do you wish to move units from?" << std::endl;
-            std::getline(std::cin,sourceTerritory);
-            auto source = std::ranges::find_if(allTerritories, [&sourceTerritory](Territory* territory) {
-                return territory->getName() == sourceTerritory;
-            });
-
-
-            std::cout << "To what territory should these units be deployed?" << std::endl;
-            std::getline(std::cin,targetTerritory);
-            auto target = std::ranges::find_if(allTerritories, [&targetTerritory](Territory* territory) {
-                return territory->getName() == targetTerritory;
-            });
-
-            std::cout << "Lastly, how many units do you want to move." << std::endl;
-            std::cin >> numOfUnits;
-            std::cin.ignore(std::numeric_limits<std::streamsize>::max(), '\n');
-
-            if (source != allTerritories.end() && target != allTerritories.end()) {
-                player->getOrdersList()->addOrder(std::make_unique<AdvanceOrder>(player, *source, *target, new int(numOfUnits)));
-                chosenOrderIndex = 0;
-            } else {
-                std::cout << "The target or source territory is not in the list of territories to defend or attack. Try again" << std::endl;
-            }
-        } else if (orderInput == stringEnumCards[1]) {
-            std::string sourceTerritory;
-            std::string targetTerritory;
-            int numOfUnits = 0;
-
```

Comment:
Keep this stuff and everything else that depended on it because there is still a possibility of having an AdvanceOrder card in the hand